libraries/testing/include/eosio/testing/tester.hpp

```diff
@@ -152,14 +152,14 @@ namespace eosio { namespace testing {
 
          virtual ~base_tester() {};
 
-         void              init(const setup_policy policy = setup_policy::full, db_read_mode read_mode = db_read_mode::SPECULATIVE);
+         void              init(const setup_policy policy = setup_policy::complete, db_read_mode read_mode = db_read_mode::SPECULATIVE);
          void              init(controller::config config, const snapshot_reader_ptr& snapshot);
          void              init(controller::config config, const genesis_state& genesis);
          void              init(controller::config config);
          void              init(controller::config config, protocol_feature_set&& pfs, const snapshot_reader_ptr& snapshot);
          void              init(controller::config config, protocol_feature_set&& pfs, const genesis_state& genesis);
          void              init(controller::config config, protocol_feature_set&& pfs);
-         void              execute_setup_policy(const setup_policy policy);
+         void              execute_setup_policy(const setup_policy policy, const std::vector<digest_type>& ignored_features = {});
```

Comment:
I don't like the fact that this allows you to specify policy and ignored_features independently, but ignored_features is only meaningful for setup_policy::full.

Reply:
I'm trying a different approach in my latest commit.